components/abstractions/src/main/java/com/microsoft/kiota/BaseRequestBuilder.java

```diff
@@ -1,47 +1,55 @@
 package com.microsoft.kiota;
 
+import jakarta.annotation.Nonnull;
 import java.util.HashMap;
 import java.util.Objects;
 
-import jakarta.annotation.Nonnull;
-
 /** Base class for request builders */
 public abstract class BaseRequestBuilder {
     /** Path parameters for the request */
-    @Nonnull
-    protected HashMap<String, Object> pathParameters;
+    @Nonnull protected HashMap<String, Object> pathParameters;
```

Comment:
is there a way to configure the behaviour? I'm not sure this kind of change is super helpful

Reply:
Actually, after [reading the documentation](https://github.com/diffplug/spotless/tree/main/plugin-gradle#formatannotations) I retract my objection here. This annotation qualifies the Type of the field, this is why it's inline. An annotation qualifying the field itself would be at the line above. It's different from dotnet, but the rationale makes sense. I'll just have to get used to it. Thanks for sharing the context.